content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not work affect runtime of Prisma CLI or Prisma Client at all).
 
```

Comment:
@janpio for the last bit of the sentence... is it meant to say:  "... does not affect the runtime work of Prisma CLI or Prisma Client..."?

Reply:
Removed